demos/checkboxradio/product-selector.html

```diff
@@ -0,0 +1,127 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>jQuery UI Checkboxradio - Product Selector</title>
+	<link rel="stylesheet" href="../../themes/base/all.css">
+	<script src="../../external/jquery/jquery.js"></script>
+	<script src="../../ui/core.js"></script>
+	<script src="../../ui/widget.js"></script>
+	<script src="../../ui/checkboxradio.js"></script>
+	<script src="../../ui/controlgroup.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		$( "input" ).checkboxradio();
+		$( "[name='shape']").on( "change", function(){
+			$( ".shape" ).removeClass( "circle pill square rectangle" )
+				.addClass( $( this ).val() );
+		});
+		$( ".toggle" ).on( "change", function(){
+			if ( $( this ).is( ".brand-toggle" ) ) {
+				var checked = $( this ).is( ":checked" ),
+					value = $( "[name='brand']" ).filter( ":checked" ).attr( "data-" + this.id )
+				$( ".shape" ).css( this.id, checked ? value : "" );
+			} else {
+				$( ".shape" ).toggleClass( this.id, $( this ).is( ":checked") );
+			}
+		});
+		$( "[name='brand']").on( "change", function(){
+			$( "input" ).filter( ":checked" ).not( this ).trigger( "change" );
+		});
+		$( "input" ).filter( ":checked" ).not( this ).trigger( "change" );
+		$( ".shape-bar" ).controlgroup();
+		$( ".toggles" ).controlgroup({
+			direction: "vertical"
+		});
+	});
+	</script>
+	<style>
```

Comment:
A few general suggestions:

* Smaller headers
* Uppercase "customize"
* Move the output a bit to the right and down, should have decent whitespace around it
* Make the brand a controlgroup as well
* Description doesn't mention controlgroup